showcase/src/elm/Routes/AlertComponent.elm

```diff
@@ -24,6 +24,7 @@ type alias Model =
     , descriptionExample : Container.Model () Never
     , typeExample : Container.Model () Never
     , closeableExample : Container.Model CloseableExample.Model CloseableExample.Msg
+    , version : String
```

Comment:
Since an API request can fail, I think a `Maybe String` will definitely be a better choice. I just went with strings for simplicity, and since I was hardcoding it anyway lol. 

I'll change the versioning to `Maybe String`, then introduce the extensible records in another commit.

Reply:
I wasn't thinking about API requests failing, but rather that the `version` environment variable won't exist for local development (at least, not by default??).

Reply:
Fixed the versioning issue! Interestingly, I get errors when I try to declare `Versioned` in the `DocumentationRoute` type alias. I'll leave this unresolved in case this is a problem, though I think it should be fine.

> I wasn't thinking about API requests failing, but rather that the version environment variable won't exist for local development (at least, not by default??).

Oh that's a good point. Would that mean the URL generator for local development won't work? I guess that would be a later issue, though.